bemani/backend/iidx/base.py

```diff
@@ -38,9 +38,25 @@ class IIDXBase(CoreHandler, CardManagerHandler, PASELIHandler, Base):
     CHART_TYPE_N14 = 3
     CHART_TYPE_H14 = 4
     CHART_TYPE_A14 = 5
-    # Beginner charts only save status
+    # Beginner charts only save status(not anymore lol)
```

Comment:
Should be updated to reflect that Beginner charts only save status up though <release>